source/Nevermore/LikeComparisonExtensions.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Nevermore
+{
+    public static class LikeComparisonExtensions {
+        public static string EscapeForLikeComparison(this string value)
+        {
+            return value.Replace("[", "[[]").Replace("%", "[%]");
```

Comment:
Underscore is also a wildcard character, per [the docs](https://learn.microsoft.com/en-us/sql/t-sql/language-elements/like-transact-sql?view=sql-server-ver16#use-wildcard-characters-as-literals). Perhaps we should handle that too?

Reply:
Good catch, I've added a case for this 👍